envbuilder.go

```diff
@@ -625,6 +625,10 @@ func Run(ctx context.Context, options Options) error {
 
 		endStage := startStage("🏗️ Building image...")
 		// At this point we have all the context, we can now build!
+		var registry_mirror []string = nil
```

Comment:
To idiomatically match our other var structures, let's initialize this as:
```
registryMirror := []string{}
```

Reply:
Fixed.